packages/app/src/cli/services/generate.ts

```diff
@@ -133,6 +138,21 @@ function buildGenerateOptions(
       app,
       extensionType: extensionContent.specification.identifier,
       cloneUrl: options.cloneUrl,
+      extensionConfig: {
```

Comment:
What was this used for @omnisyle?

Reply:
this was here because we re-used the generate extension function for migration which makes them use the same templates. Therefore we had to give some defaults for new extensions